code/modules/food_and_drinks/kitchen_machinery/smartfridge.dm

```diff
@@ -42,11 +42,18 @@
 	update_icon()
 
 /obj/machinery/smartfridge/update_icon()
-	var/startstate = initial(icon_state)
 	if(!stat)
-		icon_state = startstate
+		switch(contents.len)
+			if(0)
+				icon_state = "[initial(icon_state)]"
+			if(1 to 20)
+				icon_state = "[initial(icon_state)]1"
+			if(21 to 40)
+				icon_state = "[initial(icon_state)]2"
+			if(41 to INFINITY)
```

Comment:
maybe increse these numbers? roundstart fridge already has enough items to have icon[2] and a good fridge has 200+ pills

Reply:
roundstart fridge has 19 items

I can probably increase this a bit although this used for more than just chemistry and high numbers don't matter so long as you can tell the machine has been stocked

Reply:
I like it if the chem fridge does start close to the next icon threshold, so you can tell from a glance if the chemists have put anything there at all. Last stage needing more items sounds good though.

Reply:
roundstart should be on icon1